examples/raspberry-pi-threads.rs

```diff
@@ -0,0 +1,58 @@
+use navigator_rs::{Navigator, UserLed};
+use std::sync::{Arc, Mutex, RwLock};
+use std::thread::{self, sleep};
+use std::time::Duration;
+
+fn main() {
+    println!("Creating your SensorData cache!");
+    let sensor_data_lock = Arc::new(RwLock::new(navigator_rs::SensorData::default()));
```

Comment:
We usually don't put suffixes like '_lock', or '_mutex'. The type system tells you already what it is, and there's no need here to differentiate 'sensor_data' from 'sensor_data_lock'. Clippy warns about '_enum' and '_struct' suffixes, this is the same idea.

I suggest you rename all variables in this code.